x/programs/cmd/simulator/cmd/program.go

```diff
@@ -146,9 +154,20 @@ type ProgramStore struct {
 	state.Mutable
 }
 
-func (s *ProgramStore) GetProgramBytes(ctx context.Context, program codec.Address) ([]byte, error) {
-	// TODO: take fee out of balance?
-	programBytes, exists, err := GetProgram(ctx, s, program)
+func (s *ProgramStore) GetAccountProgram(ctx context.Context, account codec.Address) (ids.ID, error) {
+	programID, exists, err := getAccountProgram(ctx, s, account)
+	if err != nil {
+		return ids.Empty, err
+	}
+	if !exists {
+		return ids.Empty, errors.New("unknown account")
+	}
+
+	return programID, nil
+}
+
+func (s *ProgramStore) GetProgramBytes(ctx context.Context, programID ids.ID) ([]byte, error) {
+	programBytes, exists, err := getProgram(ctx, s, programID)
```

Comment:
Should we back off the TODO ? It doesn't seem to be resolved yet:
```suggestion
	// TODO: take fee out of balance?
	programBytes, exists, err := getProgram(ctx, s, programID)
```